fachhochschule-vorarlberg-note.csl

```diff
@@ -28,38 +28,54 @@
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
   <locale xml:lang="de">
+    <style-options punctuation-in-quote="true"/>
     <terms>
       <term name="et-al">u.a.</term>
-      <term name="anonymous" form="short">o.&#160;A.</term>
-      <term name="no date" form="short">o.&#160;J.</term>
+      <term name="anonymous" form="short">o. A.</term>
+      <term name="no date" form="short">o. J.</term>
```

Comment:
@rmzelle yes it was intentional, because the other Fachhochschule Vorarlberg Style also replaced them.